nbs/models.moment.ipynb

```diff
@@ -0,0 +1,1082 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    class PositionalEmbedding(nn.Module):
</pre><p>Can we reuse the class in common_modules?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1011/discussion/#comment-1691400404'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
It's not the same, the MOMENT version requires an additional input and selection mechanism in the forward method depending on the model name....